src/vue/components/panel-compact.vue

```diff
@@ -1,187 +1,104 @@
 <template>
-  <div :class="rootClasses" :data-notification-visible="data && data.notification && data.notification.important"
-       :style="mainstyles" class="alpheios-panel auk"
+  <div :class="rootClasses" :data-notification-visible="$store.state.ui.notification.visible && $store.state.ui.notification.important"
+       :style="mainstyles" class="alpheios-panel alpheios-panel--compact auk"
        data-component="alpheios-panel"
        data-resizable="true" id="alpheios-panel-inner" v-on-clickaway="attachTrackingClick"
```

Comment:
I am not sure if we want the clickAway directive enabled for the mobile ui. I think probably not.

Reply:
Removed it